dallinger/command_line/__init__.py

```diff
@@ -523,49 +508,55 @@ def _current_hits(service, app):
+def hits(app, sandbox, recruiter):
+    """List all HITs for the recruiter account or for a specific experiment id."""
     if app is not None:
         verify_id(None, "--app", app)
-    formatted_hit_list = []
-    dateformat = "%Y/%-m/%-d %I:%M:%S %p"
-    for h in _current_hits(_mturk_service_from_config(sandbox), app):
-        title = h["title"][:40] + "..." if len(h["title"]) > 40 else h["title"]
-        description = (
-            h["description"][:60] + "..."
-            if len(h["description"]) > 60
-            else h["description"]
-        )
-        formatted_hit_list.append(
-            [
-                h["id"],
-                title,
-                h["annotation"],
-                h["status"],
-                h["created"].strftime(dateformat),
-                h["expiration"].strftime(dateformat),
-                description,
-            ]
-        )
-    out = Output()
-    out.log("Found {} hit[s]:".format(len(formatted_hit_list)))
-    out.log(
-        tabulate.tabulate(
-            formatted_hit_list,
-            headers=[
-                "Hit ID",
-                "Title",
-                "Annotation (experiment ID)",
-                "Status",
-                "Created",
-                "Expiration",
-                "Description",
-            ],
-        ),
-        chevrons=False,
-    )
+
+    rec = by_name(recruiter, skip_config_validation=True)
+    rec.hits(app, sandbox)
+
+
+@dallinger.command()
+@click.option("--hit_id", default=None, help="MTurk HIT ID")
+@click.option(
+    "--sandbox",
+    is_flag=True,
+    flag_value=True,
+    help="Look for HITs in the MTurk sandbox rather than the live/production environment",
+)
+@click.option("--recruiter", default="mturk", help="Experiment id")
+def hit_details(hit_id, sandbox, recruiter):
+    """Print the details of a specific HIT for a recruiter."""
+    rec = by_name(recruiter, skip_config_validation=True)
+    details = rec.hit_details(hit_id, sandbox)
+    print(json.dumps(details, indent=4, default=str))
+
+
+@dallinger.command()
+@click.option("--hit_id", default=None, help="MTurk HIT ID")
+@click.option(
+    "--sandbox",
+    is_flag=True,
+    flag_value=True,
+    help="Look for HITs in the MTurk sandbox rather than the live/production environment",
+)
+@click.option("--recruiter", default="mturk", help="Experiment id")
+@click.option("--path", default=None, help="Filename/path for the qualification file")
+def copy_qualifications(hit_id, sandbox, recruiter, path):
+    """Copy qualifications from an existing HIT and save them to a JSON file."""
+    rec = by_name(recruiter, skip_config_validation=True)
+    if path is None:
+        path = rec.default_qualification_name
+    assert path.endswith(".json"), "Qualification path must be a json file"
+    if exists(path):
+        if not click.confirm(f"Overwrite existing qualification file: {path}?"):
+            click.echo("Aborting...")
+            return
```

Comment:
Instead of 

```py
            click.echo("Aborting...")
            return
```

I think we want

```py
            click.abort()
```

Reply:
I copy pasted this from there: 
![image](https://user-images.githubusercontent.com/19990643/226859268-d9152001-85f0-4e3b-84c2-f71af36648b7.png)

I can't find a single instance of `click.abort()` in the Dallinger source code, so I would prefer to keep it as it is.